### ISSUE TITLE
error: cannot find symbol for MavenWrapperMain

### ISSUE THREAD

**Issue by philip-khor** [KonduitAI/konduit-serving#181]
I'm getting `error: cannot find symbol` when the MavenWrapperMain class is imported by the Maven Wrapper script for 
- File 
- FileInputStream
- FileOutputStream
- URL
- IOException 
- Authenticator
- ReadableByteChannel
- Channels

See Dockerfile and trace at https://gist.github.com/philip-khor/339df906418a8c0355b1bb0e808f6d9b

This is possibly caused by several import statements removed from [MavenWrapperDownloader.java](https://github.com/KonduitAI/konduit-serving/blob/master/.mvn/wrapper/MavenWrapperDownloader.java) in b247b21. 

```java
import java.net.*;
import java.io.*;
import java.nio.channels.*;
```

This error disappears when `mvn -N io.takari:maven:0.7.7:wrapper` is run before `mvnw`.

**Comment by philip-khor** [comment]
This is not noticeable on certain Linux installations because the `mvnw` script first checks if wget or curl is available; if not it uses the Java script to build the file.

**Comment by philip-khor** [comment]
Closed in #185
